user-manual.md

```diff
@@ -0,0 +1,47 @@
+# A high level description of RecipeCart:
+This system is a website for uploading, searching, and rating recipes. Using our descriptive recipe tagging system (missing), users can specify dietary restrictions or preferences that match the recipes they wish to search for. Users can also leave ratings (missing) on recipes to let others know the average quality of the recipe.  
+
+# How to install the software:
+To install the software, there are separate requirements for the frontend and the backend. After cloning the repo to your local system, follow the steps below:
+- Frontend:
+    - Install Node.js (v18) from the Node website
+    - In the `/front-end/` directory, run `npm i` to install dependencies
+- Backend:
+    - 
+
+# How to run the software:
+After installing all the necessary dependencies, follow the steps below to run the software:
+- Frontend:
+    - In the `/front-end/` directory, run `npm run build`
+    - Run `npm run start` to launch the website
+    - Access the website at `http://localhost:8080`
+- Backend:
+    - 
+
+# How to use the software:
+Once the dependencies are installed and the website/backend are running, here is the functionality you can utilize:
+- Recipe Search
+    - Search the database for recipes using the search bar
+    - Filter by a variety of tags (WIP)
+- Add Recipes to the Database
+    - After logging in (WIP), add recipes to the local Database using the Add Recipe page (WIP)
+- Save a Recipe
+    - After logging in, save a recipe to visit it later through the Saved Recipes page (WIP)
+- Rate a Recipe
+    - After logging in, rate a recipe from your Saved Recipes page (WIP)
+
+
+# How to report a bug:
+Currently, we are using GitHub Issues to track known bugs. To report a new bug, simply navigate to the Issues tab on the RecipeCart repository and create a new issue.
+
+Information required to report a bug:
+- Description of the bug
+- Where did the bug happen?
+- Expected outcome
+- Actual outcome
+- Steps to reproduce the unexpected outcome
+- Any files they want to attach such as visual outcome of the bug
```

Comment:
You should eventually use either [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) or [issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository). Okay to not have this done by tonight but ideally ready when peer review starts.